PlayerControls/templates/EnumPrism.stencil

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+{% macro associatedValueVarMultiType case %}({% for associatedValue in case.associatedValues %}{% if associatedValue.localName != nil %}{{ associatedValue.localName }}: {% endif %}{{ associatedValue.typeName }}{% if not forloop.last %}, {% endif %}{% endfor %})?{% endmacro %}
```

Comment:
As nobody reads results - optimise it for reading, rather than optimising for the generated format.